types/imap/index.d.ts

```diff
@@ -83,6 +83,8 @@ declare namespace Connection {
             /** (Only available with status() calls) Number of messages in this mailbox not having the Seen flag (marked as not having been read). */
             unseen: number;
         };
+        /** (Available with openBox() and status()) The highest modification sequence value of all messages in the mailbox. */
+        highestmodseq: string;
```

Comment:
The comment seems to imply that this isn't always present. Should this be `highestmodseq?: string | undefined`? Or is this actually always present? (The upstream code isn't very clear.)

Reply:
If the server supports the persistent storage of mod-sequences it should return it as documented in RFC4551. And it would appear that the box object can be returned only with these two functions. But it was in the update of imap lib so I kept this formulation.

Reply:
I don't know anything about the RFC, I'm only referring to the types. Basically, I'm just trying to verify that it's the case that in every place that a function returns a `Box`, `highestmodseq` is set with a string value.

Reply:
For me, this is the case.